src/context/VerifyContext.js

```diff
@@ -58,8 +58,10 @@ export function VerifyProvider(props) {
   /* load captures when the page or page size changes */
   useEffect(() => {
     const abortController = new AbortController();
-    setCaptureImages([]);
-    loadCaptureImages({ signal: abortController.signal });
+    if (!isLoading) {
```

Comment:
This doesn't fix the problem – I'm still seeing the same behaviour.
If it did prevent the error, it would result in the wrong captures showing when the page changes during a load.